### DISCUSSION TITLE
Webhook for triggering a specific Docker container:

### DISCUSSION THREAD

**Discussion by nihilo** [directus/directus#23275] [Feature Requests]
### Summary

I want to use a trigger, when the condition “status: draft” is reached for a specific collection, to trigger a via webhook a specific docker container. The container needs the collection name in order to process just the data from this specific collection.

### Motivation

My webhook url is: http://192.168.0.37:5000/run-docker

This are my Condition Rules:

```
{
    "$trigger": {
        "event": {
            "_eq": "Motion_data.items.update"
        },
        "payload": {
            "status": {
                "_eq": "draft"
            }
        },
        "collection": {
            "_eq": "Motion_data"
        }
    }
}

```

The condition options transferred to the webhook node is in the logs this -> 
```
{
  "filter": {
    "$trigger": {
      "event": {
        "_eq": "Motion_data.items.update"
      },
      "payload": {
        "status": {
          "_eq": "draft"
        }
      },
      "collection": {
        "_eq": "Motion_data"
      }
    }
  }
}


```
but the options for the webhook it self is:
```
{
  "url": "http://192.168.0.37:5000/run-docker",
  "body": "{\n  \"keys\": \"undefined\",\n  \"event\": \"undefined\",\n  \"payload\": {\n    \"status\": \"undefined\"\n  },\n  \"collection\": \"undefined\"\n}\n",
  "method": "POST",
  "headers": null
}

```

### Detailed Design

.

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

`{
  "url": "http://192.168.0.37:5000/run-docker",
  "body": "{\n  \"keys\": \"undefined\",\n  \"event\": \"undefined\",\n  \"payload\": {\n    \"status\": \"undefined\"\n  },\n  \"collection\": \"undefined\"\n}\n",
  "method": "POST",
  "headers": null
} `

### Alternatives

.

### Adoption Strategy

.

### Unresolved Questions

Any ideas or tips how i can solve this?

**Comment by hanneskuettner** [comment]
This does not seem to be a feature request but rather a usage question? Can you please direct those towards our awesome community at https://chat.directus.io
